### DISCUSSION TITLE
Context for High data transfer between argocd application controllers and redis

### DISCUSSION THREAD

**Discussion by Debraj-git** [argoproj/argo-cd#13344] [Q&A]
While debugging our data transfer charges we found the below services to be major contributors.

Need to understand the context of communication (what is actually communicated between them) for the below services.
1. Communication between argocd application controllers and redis
2. Communication between argocd repository servers and argocd application controllers

Thanks in advance!

**Comment by jfavellar90** [comment]
I experienced a similar issue. We have an Argo CD instance in EKS and we noticed a high inter-AZ data transfer. We activated VPC [flow logs](https://docs.aws.amazon.com/vpc/latest/userguide/flow-logs-s3.html#flow-logs-s3-create-flow-log) and found records sending ~80MB to ArgoCD Redis instance in a single transaction.

It would be interesting to understand better how Argo CD pieces communicate with each other to get a workaround. For now, we'll try to change to Redis HA to check if the issue is mitigated.
